src/datastore/database.cpp

```diff
@@ -215,3 +168,102 @@ void Database::saveToFile(const std::string& path)
     // Call saveData to store data at the path provided
     saveData(path, allRows);
 }
+
+/**
+ * @brief Loads distances as a tuple
+ *
+ * This functions takes a path, extracts all the data and instantiates
+ * tuples of distances std::tuple<int,int,int>(fromStadiumId,toStadiumId,Distance);
+ *
+ * @param filepath path to .csv data for teams,stadiums,and souvenirs
+ */
+void Database::loadDistancesFromFile(const std::string& filepath)
+{
+    /* Load distance data from file */
+    std::vector<std::vector<std::string>> distanceData = loadData(filepath);
+    /* ForEach row of data create a tuple std::tuple<int,int,int>(fromStadiumId,toStadiumId,Distance); */
+    for(const std::vector<std::string>& distance : distanceData)
+    {
+        distances.push_back(std::tuple<int,int,int>(std::stoi(distance[0]),
+                                                    std::stoi(distance[2]),
+                                                    std::stoi(distance[4])));
+    }
+}
+
+/**
+ * This method returns all the teams
+ *
+ * @return teams is returned
+ */
+nstd::map<int,Team> Database::getTeams()
```

Comment:
Why were all these functions moved from their original position?

Reply:
Moved the methods to be in the same order as the header file